include/libopencm3/sam/4l/memorymap.h.m4

```diff
@@ -0,0 +1,97 @@
+m4_divert(-1)
+m4_dnl Single memory address, defined via
+m4_dnl name, base, offset
+m4_define(`MEM_NAME', `m4_ifelse(`$1', `RESERVED', `/* ($2 + $3) RESERVED */',
```

Comment:
You want to edit this commit to drop them, not add a later commit that drops them out after adding them.